Lizard Virus

Profakos opened the pull request:
Made by Collen, fixed by me. Turns the afflicted person into a lizard.
Adds a bottle too, for shenanigans.

Profakos committed:
Lizard Virus

Made by Collen, fixed by me. Turns the afflicted person into a lizard.
Adds a bottle too, for shenanigans.

Giacom commented on code/datums/diseases/advance/symptoms/lizardification.dm:
```diff
@@ -0,0 +1,48 @@
+						H.dna.mutantrace = "lizard"
+						H.update_body()
+						H.update_hair()
+						A.cure()
```
That's going to cure them of all their other symptoms; just have it so you check if they're a lizard or not in the 5th stage after they are turned.

Another possible feature, that you can add, is that when the symptom dies you can use the symptom proc "end()" to reverse the lizard form.

Profakos commented:
Collen said that the first bit is intentional, once the virus did its job, it goes away completely. However, moving the selfcure part out of the fifth stage is a good idea.

On the other hand, she did it this way because neither I or she knew how to cure results of the symptons when the virus goes away. Thank you for the help! I'll replace self-cure with that.

Profakos committed:
Healing Changes

Virus doesn't remove itself when the transformation is complete.
When cured, the human will become a human again. Alternatively, someone
who was a a lizard all along becomes human.
Adds the ticked in DME as I promised.

Profakos commented:
Again, thank you! Now we know how to make viruses that heal a lasting change they caused.

Giacom commented:
You should check they aren't a lizard already before displaying the symptom messages.

Profakos committed:
Lizard won't try to turn into lizards

Profakos commented:
Alright, moved the check before the message display. Now lizards won't try to turn into lizards (neither will other mutanttraces).

Profakos committed:
Changed  the .dme as Sayu requested
